services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -302,25 +326,25 @@
   [request-id params]
   (->> (metadata/get-permanent-id-request request-id)
        parse-service-json
-       (format-perm-id-req-response (:shortUsername current-user))))
+       (format-permanent-id-request-details (:shortUsername current-user))))
 
 (defn admin-list-permanent-id-requests
   [params]
   (-> (metadata/admin-list-permanent-id-requests params)
       parse-service-json
-      (update-in [:requests] format-perm-id-req-list)))
+      (update :requests format-perm-id-req-list)))
 
 (defn admin-get-permanent-id-request
   [request-id params]
   (->> (metadata/admin-get-permanent-id-request request-id)
        parse-service-json
-       (format-perm-id-req-response (:shortUsername current-user))))
+       (format-permanent-id-request-details (:shortUsername current-user))))
 
 (defn update-permanent-id-request
   [request-id params body]
   (let [response (->> (metadata/update-permanent-id-request request-id body)
                       parse-service-json
-                      (format-perm-id-req-response (:shortUsername current-user)))]
+                      (format-permanent-id-request-details (:shortUsername current-user)))]
     (send-update-notification response)
     response))
```

Comment:
This isn't something you're changing here, but am I missing something or is `params` not used in this function? If so, why is it included?